contracts/test/foundry/FleekERC721/AccessPoints/AccessPointsAutoApprovalOff.t.sol

```diff
@@ -16,11 +17,12 @@ contract Test_FleekERC721_AccessPoint is Test_FleekERC721_Base {
         string memory score,
         string memory nameVerified,
         string memory contentVerified,
-        address owner
+        address owner,
+        string memory status
```

Comment:
Maybe we could use the enum instead of a string to make the tests more readable.
```suggestion
        FleekERC721.AccessPointStatus status
```

Reply:
Maybe. I prefer Enums because they limit the input and define the type but I see where you are coming from.

Will change